### ISSUE TITLE
Training script of nll.

### ISSUE THREAD

**Issue by Chunngai** [nusnlp/geccl#2]
Hello. Thanks for your work and kindly releasing the code. Can you also provide the fine-tuning script of NLL except CL- and CL?

**Comment by MichaelCaohn** [comment]
Hi, 

Sorry, I did not save the fine-tuning script for NLL. As long as I can recall:

For GEC-BART, we used the fine-tuning script provided in https://github.com/Katsumata420/generic-pretrained-GEC/blob/master/BART-GEC/train.sh , but in order to fit in V100 GPU, we modified the MAX_TOKENS=4000 to 1400 and changed the UPDATE_FREQ from 1 to 4. 

For GEC-PD, it's similar to the script for CL and CL-, but you need to change the criterion from label_smoothed_cross_entropy to new_max_margin_loss, and also change the data path to the data used for NLL.

To generate the data used for NLL used in GEC-BART and GEC-PD, you could refer to the preprocessing script for GEC-BART in https://github.com/Katsumata420/generic-pretrained-GEC/tree/master/BART-GEC and for GEC-PD in https://github.com/butsugiri/gec-pseudodata .

**Comment by MichaelCaohn** [comment]
Hi,

Sorry, I did not save the fine-tuning script for NLL. As long as I can recall:

For GEC-BART, we used the fine-tuning script provided in https://github.com/Katsumata420/generic-pretrained-GEC/blob/master/BART-GEC/train.sh , but in order to fit in V100 GPU, we modified the MAX_TOKENS=4000 to 1400 and changed the UPDATE_FREQ from 1 to 4.

For GEC-PD, we reported the NLL result used in the CWEB dataset paper (https://aclanthology.org/2020.emnlp-main.680.pdf).
